### ISSUE TITLE
Declutter noob screen

### ISSUE THREAD

**Issue by orbitmechanic** [orbitmechanic/USAMobile#153]
<img width="855" alt="Screen Shot 2021-11-16 at 10 24 36 PM" src="https://user-images.githubusercontent.com/56613757/142129721-8567d124-1c81-404f-a228-324124f65db7.png">
So the above is the current new-user/browser experience.  There are things here that shouldn't be:

- Top line: the expert stage on/off switch should not be here.  The expert stage needs to default to visible, only dismissable after the user has 1.) Added Metamask and 2.) Added Polygon.  Only then should they be able to dismiss the expert stage.
- Top line: The "Add polygon to MetaMask" button (infinity/laundry symbol) should not display when we know there's no MetaMask yet.
- If the "Click Here to Install Metamask" button is visible the whole top row of buttons should be the second row.  It's too wide with "USA Wallet" on the same line.  The "Top Row" should just include dark/light mode switch, the "Logout" and the "Click Here to Install Metamask" button.  And all that should be the second line after the hulking "USA Wallet" banner.
- Middle line: All four mode buttons should be gone before MetaMask and Polygon is installed.  The only thing a user should be able to do at first is install Metamask, then Polygon THEN the mode buttons become visible.
- Middle Line: as before: if the user has Metamask and Polygon installed but has nothing in their portfolio the only two buttons that should show are "Buy/Sell" and "Send/Receive". And those buttons should be reduced to "Buy" and "Receive" respectively because that's all they can do with no crypto.
- Bottom line: the Buy Crypto/Transak widget should indeed show up as the default, but only after MetaMask and Polygon are installed.
- We may need a second patriotic banner picture for these early states to fill all the blank space this new logic will create below the expert stage.

**Comment by orbitmechanic** [comment]
<img width="455" alt="Screen Shot 2021-11-19 at 9 09 25 AM" src="https://user-images.githubusercontent.com/56613757/142636257-0a30f8a1-77d9-4d5a-bd77-56d362821e5c.png">
Looks much better.

**Comment by orbitmechanic** [comment]
<img width="464" alt="Screen Shot 2021-11-19 at 9 12 43 AM" src="https://user-images.githubusercontent.com/56613757/142636549-90615de1-6127-47e2-9e74-9485de1e8702.png">
First state looks great!

**Comment by orbitmechanic** [comment]
<img width="481" alt="Screen Shot 2021-11-19 at 9 13 45 AM" src="https://user-images.githubusercontent.com/56613757/142636643-4a9253e9-1d77-44eb-8985-0e22e79cb38f.png">
First state looks great!

**Comment by orbitmechanic** [comment]
<img width="464" alt="Screen Shot 2021-11-19 at 9 12 43 AM" src="https://user-images.githubusercontent.com/56613757/142636678-04ef5615-f610-4820-abbe-afad3ca6fd03.png">
I like how the button line drops down when the MetaMask button gets crazy wide.  Well done.
Still needs two things in this state tho:

- The expert dialog needs to switch to "Click above to install MetaMask in your browser.
- I need to give you a new picture to replace Lady Liberty...because she still says "Press Log In to Begin" and we're clearly logged into Moralis at this point.

**Comment by orbitmechanic** [comment]
Please use this image instead of the Statue of Liberty for the second state (logged into Moralis, but no MetaMask):
<img width="586" alt="Screen Shot 2021-11-19 at 9 37 42 AM" src="https://user-images.githubusercontent.com/56613757/142640559-cb757c54-9d2f-4b65-9577-7808a3357cfa.png">

**Comment by orbitmechanic** [comment]
(I can't resist sneaking in a rocket SOMEWHERE.  :P )

**Comment by orbitmechanic** [comment]
Then there will be a third noob state: signed into Moralis, Metamask installed--but no Polygon yet.  

- Keep Uncle Sam up.
- Change dialog to "Add Polygon Network to Metamask to discount transaction fees.
- Use the following splash screen:

<img width="510" alt="Screen Shot 2021-11-19 at 9 54 21 AM" src="https://user-images.githubusercontent.com/56613757/142643251-d1fd4839-0b4d-47d0-9761-5d11c4bca3a0.png">

**Comment by orbitmechanic** [comment]
And then finally when we're all set up on our end:
Leave Uncle Sam up.
Dialog switched to "Congratulations!  You are ready to use cryptocurrency!"
Splash image switch to:
<img width="620" alt="Screen Shot 2021-11-19 at 10 04 24 AM" src="https://user-images.githubusercontent.com/56613757/142644650-87de2e8d-ecea-4253-9242-8e4a089a2408.png">

**Comment by orbitmechanic** [comment]
Ok so there's this odd state wherein I've authenticated into Moralis using MetaMask, yet MetaMask itself is not connected to the website yet.  In this state the website can't see my portfolio and presumes I have nothing in it. Accordingly it does the right thing and dumps me into the buy crypto screen.  However, once I connect MetaMask to the website it remains on the buy screen.  The check for portfolio emptiness needs to be re-run on MetaMask<=>Website connection events.  It should switch me to the portfolio page once it detects my balances.
![Screen Shot 2021-11-22 at 9 50 14 AM](https://user-images.githubusercontent.com/56613757/142882273-7670f95d-ee8d-435c-b4e4-e92e80441c00.png)

**Comment by orbitmechanic** [comment]
![Screen Shot 2021-11-23 at 8 15 30 AM](https://user-images.githubusercontent.com/56613757/143030929-2f1a552e-8d93-4731-becb-43a063945753.png)
I got into this situation by switching my MetaMask from Polygon back to Ethereum MainNet.  I've a portfolio there on this address.  I can't see it.  The mode buttons are gone.  And I'm not getting orders to switch back to Polygon.
